config/jobs/kubernetes/minikube/minikube.yaml

```diff
@@ -0,0 +1,30 @@
+presets:
+- label:
+    preset-minikube-e2e-creds: "true"
+  env:
+  - name: GCS_CREDS
+    value: /root/.config/gcloud/k8s-minikube-build-gcs.json
+  volumes:
+  - name: k8s-minikube-build-gcs
+    secret:
+      secretName: k8s-minikube-build-gcs
+  volumeMounts:
+  - name: k8s-minikube-build-gcs
+    mountPath: /root/.config/gcloud
+    readOnly: true
+
+presubmits:
+  kubernetes/minikube:
+  - name: pull-linux-kvm
```

Comment:
Please change the job name to reference minikube (maybe `pull-minikube-linux-kvm`?) as these names are globally scoped.